.travis.yml

```diff
@@ -1,17 +1,21 @@
 language: node_js
 os:
   - linux
+  - osx
```

Comment:
Do you know if there's a way to disable these selectively for PRs? On the free tier it takes OSX a long time to run which adds additional clicks to check whether something passed on Linux.

Reply:
Actually, OSX builds are now a lot faster as compared to earlier in the year. I don't know how to disable them selectively in any case.

Reply:
@Tyriar You can see [here](https://travis-ci.org/Tyriar/node-pty/builds/363522921?utm_source=github_status&utm_medium=notification) that the builds with OSX last about 15 minutes. Not only that, the OSX builds are all faster than the Linux builds.